test/misc.jl

```diff
@@ -1402,3 +1402,13 @@ end
     GC.gc(true); yield()
     @test in_fin[]
 end
+
+@testset "Base docstrings" begin
+    undoc = Docs.undocumented_names(Base)
+    @test_broken isempty(undoc)
+    @test undoc == [:BufferStream, :CanonicalIndexError, :CapturedException, :Filesystem, :IOServer, :InvalidStateException, :Order, :PipeEndpoint, :Sort, :TTY]
```

Comment:
Test is failing on Windows, which has an additional undocumented `OS_HANDLE` symbol.

Reply:
Nevermind, this is passing now that I added the `WindowsRawSocket` docstring.

Reply:
What confuses me is that `stream.jl` defines `const OS_HANDLE = WindowsRawSocket`, and this PR added a docstring for `WindowsRawSocket` to `libc.jl` precisely to fix this failure.

Why is it still showing up as undocumented?

Reply:
Ah, I see — apparently the docstring only works if it is defined at the top level.

Reply:
I think using `@doc` explicitly should fix the problem.